jenkins_jobs/update_versions_readme/script.sh

```diff
@@ -0,0 +1,9 @@
+python host-os.py \
+  --verbose \
+  update-versions-readme \
```

Comment:
> IMHO it is better to automatically update README whenever the yamls are updated to avoid extra work and accidental inconsistencies.

That's the idea.
Whenever there is a Merge event on the `versions` repository, we trigger an `update-versions` job. Hence my comment on updating this PR with a `trigger_*` job.

See my first comment on this PR:

> I'll update this PR with a commit creating a trigger for the GitHub Merge event, in order to
> start the update job automatically.